src/components/itemPage/Item.jsx

```diff
@@ -103,6 +142,7 @@ class Item extends Component {
       let item = data[params.corpus][params.item];
       let itemTopics = (item.topic) ? groupBy(item.topic, ['viewpoint']) : {};
       let topics=this.state.item.topic || {};
+      topics = (itemTopics.length > 0) ? topics : {};
```

Comment:
What is it for? Is it related to this REFACTORING? Or is it a FIX for a special case that was not handled?

Reply:
He's definitely part of this REFACTORING.
It fixes a noticeable problem with this update: the last deleted topic was not updated on all other client browsers.